### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kateloschinina**
Update README with the most relevant info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
-        SESSION_API_HOST:process.env.SESSION_API_HOST
-    };
+## Testing
 
-An API key for the services in this package is requried. Details on obtaining one for your application can be found in the [membership api gateway](https://developer.ft.com/docs/membership_platform_api/) guide.
+[TBD - when new tests are implemented]
 
-## Usage
+## Deployment
+This component has been created to be included throughout other KAT components.
```
Comment (dapenguin) [inline README.md]: Just for the sake of consistency, maybe check whether "component" is the correct term, or whether it should be "module".

File: README.md [hunk 2]
```diff
@@ -1,37 +1,47 @@
...
 
-In your applicaition entry point (so app.js, server.js or index.js) require the utils package, utilities.Config.js and the initCookieSession:
+### How to update a repo that uses the component to the new version
+If you want to update connected components with the latest version, you need to follow the following steps:
+1. Create a new repository release on gitHub. Please follow naming convention of previous releases.
```
Comment (dapenguin) [inline README.md]: Needs an uppercase "G" on "GitHub".
Change "follow naming" to read "follow the naming".

File: README.md [hunk 3]
```diff
@@ -1,37 +1,47 @@
 # kmt-utilities
-Common middleware for Knowledge Manager Tools (KMT) microservices
+Common middleware for KAT microservices.
 
-## Installation guide
+KAT (Knowledge & administration tools) is a part of ft.com application created for Financial Times B2B clients.
```
Comment (kateloschinina) [inline README.md]: Done in all repos.

File: README.md [hunk 4]
```diff
@@ -1,37 +1,47 @@
...
 
-Include ``"kmt-utilities": "financial-times/kmt-utilities"``as a dependency in your package.json
+## Getting started
+To get a copy of the project up and running on your local machine for development and testing purposes run `$ git clone git@github.com:Financial-Times/kmt-utilities.git`.
```
Comment (kateloschinina) [inline README.md]: Done. Is in all other repos, I guess I just have forgotten to do in this one =)
Re make - will add when make functionality will be introduced (right now is still not in place), or the person who will merge make branch in.

File: README.md (hunk 1 above)
Comment (kateloschinina) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (kateloschinina) [inline README.md]: Done.

File: README.md [hunk 5]
```diff
@@ -13,19 +13,19 @@ Use the `.env` file saved in LastPass to set up necessary environmental variable
 [TBD - when new tests are implemented]
 
 ## Deployment
-This component has been created to be included throughout other KAT components.
+This module has been created to be included throughout other KAT components.
 
-### How to update a repo that uses the component to the new version
+### How to update a repo that uses the module to the new version
 If you want to update connected components with the latest version, you need to follow the following steps:
```
Comment (dapenguin) [inline README.md]: Feels like this line should use "modules" instead of "components".

File: README.md [hunk 6]
```diff
@@ -13,19 +13,19 @@ Use the `.env` file saved in LastPass to set up necessary environmental variable
...
-1. Create a new repository release on gitHub. Please follow naming convention of previous releases.
+1. Create a new repository release on GitHub. Please follow the naming convention of previous releases.
 2. Go to `package.json` file of the component you want to update, and change `"kmt-utilities"` dependency version to the [newly released one](https://github.com/Financial-Times/kmt-utilities/releases).
```
Comment (dapenguin) [inline README.md]: Feels like this line should also use "module" instead of "component".

**Pull request merged**